docs/Syntax.md

```diff
@@ -444,7 +444,7 @@ an infinite stream of bits.
     [t1 .. t2 by n]       // Enumeration (stride)
     [t1 .. <t2 by n]      // Enumeration (stride, ex. bound)
     [t1 .. t2 down by n]  // Enumeration (downward stride)
-    [t1 .. >t2 down by n] // Enumeration (downward stride, ex. bound)
+    [t1 .. <t2 down by n] // Enumeration (downward stride, ex. bound)
```

Comment:
I think the downward one is supposed to be using `>`?  As in:
```
f = [ 3 .. > 1 down by 1]

[3, 2]
```

Reply:
Yes, you're right. I tried this on a docker nightly version of Cryptol from 2 weeks ago that did not yet have this feature and the interpreter complained so I though it was wrong. After updating to the nightly just now I see that this works.